src/components/Button-Link/Button/Button.tsx

```diff
@@ -16,6 +17,7 @@ const Button = ({
   children,
   className = "",
   size,
+  disabled,
```

Comment:
Lol. So you just put this "disabled" here, in the props, but you don't add it on the `<button>`? 😆